frontend/ui/src/pages/Channels/ChannelsMainPage/SourceInfo/index.tsx

```diff
@@ -42,7 +39,7 @@ const SourceInfo = (props: SourceInfoProps) => {
                   return (
                     <li key={channel.sourceChannelId} className={styles.channelListEntry}>
                       <button className={styles.connectedChannelData} onClick={() => props.onChannelClick(channel)}>
-                        {channel.metadata?.imageUrl ? (
+                        {source === 'facebook' && channel.metadata?.imageUrl ? (
```

Comment:
the additional check is not needed, cause if the channel doesn't have any imageUrl it's already false